subprojects/persistent-cache/persistent-cache.gradle.kts

```diff
@@ -46,3 +46,6 @@ gradlebuildJava {
     moduleType = ModuleType.CORE
 }
 
+tasks.withType<ValidatePlugins>().configureEach {
+    enabled = false
```

Comment:
Why is this required?

Reply:
This is a temporary fix because our validate plugins task doesn't handle classloading properly. I changed a type from a concrete type to an interface and it doesn't like it. Once we have a new wrapper this must be removed